packages/esify/index.js

```diff
@@ -20,6 +20,16 @@ var TRANSFORMS = [
   {path: 'shopify-codemod/transforms/function-to-arrow'},
   {path: 'shopify-codemod/transforms/global-assignment-to-default-export', test: false},
   {path: 'shopify-codemod/transforms/global-reference-to-import'},
+  {path: 'shopify-codemod/transforms/assert/assert-to-assert-ok', test: true},
+  {path: 'shopify-codemod/transforms/assert/negated-assert-ok-to-assert-not-ok', test: true},
```

Comment:
I think we should add some comments in here, that will probably be the best way to prevent random changes in order.